sdk/src/client/bindings/java/README.md

```diff
@@ -8,8 +8,8 @@ minimum Java version >= 8
 
 ## Use in your Android project (Android Studio)
 
-1. Download the `iota-client-1.0.0-rc.3.jar` file from the [GitHub release](https://github.com/iotaledger/iota.rs/releases/tag/iota-client-java-1.0.0-rc.3) and add it as a library to your project.
-2. Download the `iota-client-1.0.0-rc.3-android.zip` file from the [GitHub release](https://github.com/iotaledger/iota.rs/releases/tag/iota-client-java-1.0.0-rc.3), unzip it and add the `jniLibs` folder with its contents to your Android Studio project as shown below:
+1. Download the `iota-client-1.0.0-rc.3.jar` file from the [GitHub release](https://github.com/iotaledger/iota-sdk/releases/tag/iota-client-java-1.0.0-rc.3) and add it as a library to your project.
+2. Download the `iota-client-1.0.0-rc.3-android.zip` file from the [GitHub release](https://github.com/iotaledger/iota-sdk/releases/tag/iota-client-java-1.0.0-rc.3), unzip it and add the `jniLibs` folder with its contents to your Android Studio project as shown below:
```

Comment:
We didn't have a client release in the sdk repo, so the links are broken

Reply:
Yeah I know, I asked myself that, but I'd say it's not a big deal if they are temporarily broken. We'll need to sort all that before the release anyway, but we should stop forwarding them to iota.rs and wallet.rs